src/Files.App/App.xaml.cs

```diff
@@ -122,6 +122,7 @@ private IServiceProvider ConfigureServices()
 				.AddSingleton<IThreadingService, ThreadingService>()
 				.AddSingleton<ILocalizationService, LocalizationService>()
 				.AddSingleton<ICloudDetector, CloudDetector>()
+				.AddSingleton<IAddItemTypesService, AddItemTypesService>()
```

Comment:
```suggestion
				.AddSingleton<IAddItemService, AddItemService>()
```
What does "types" indicate?

Reply:
File type like excel, power point, txt (all the selectable items except for folders, shortcuts and empty files)

Reply:
Would `AddItem` be a little better since it's more generic?